src/main/store.ts

```diff
@@ -2,11 +2,13 @@ import Store from "electron-store";
 
 type TypedStore = {
   poetryOptionalGroups: string[];
+  setupStarted: number;
 };
 
 export const store = new Store<TypedStore>({
   defaults: {
     poetryOptionalGroups: [],
+    setupStarted: 0,
```

Comment:
Should this be in store? it doesn't ever need to be persisted, and every time you use it you need to set it to 0 again after, I feel like it could just be a value you attach to the `global` object instead at startup, what do you think?

Reply:
You're right, attaching to global object makes more sense. Thanks